Python/auto_update_quarto.py

```diff
@@ -0,0 +1,35 @@
+"""
+This script will download and install the latest version of Quarto.
+Should work on Ubuntu >18.04.
```

Comment:
```suggestion
Should work on Ubuntu >18.04 (on machines with Intel Chips).
```
Yes, there are people that install Ubuntu on ARM macs 😬